Minor updates to access reviews(Beta) (#12973)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewinstance.md

| errors | [accessReviewError](accessreviewerror.md) collection| Collection of errors in an access review instance lifecycle. Read-only. |
| reviewers   |[accessReviewReviewerScope](../resources/accessreviewreviewerscope.md) collection| This collection of access review scopes is used to define who are the reviewers. Supports `$select`. For examples of options for assigning reviewers, see [Assign reviewers to your access review definition using the Microsoft Graph API](/graph/accessreviews-scope-concept).|
| fallbackReviewers   |[accessReviewReviewerScope](../resources/accessreviewreviewerscope.md) collection| This collection of reviewer scopes is used to define the list of fallback reviewers. These fallback reviewers will be notified to take action if no users are found from the list of reviewers specified. This could occur when either the group owner is specified as the reviewer but the group owner does not exist, or manager is specified as reviewer but a user's manager does not exist. Supports `$select`.|

> | contactedReviewers   |[accessReviewReviewer](../resources/accessreviewreviewer.md) collection| This is a read-only property. In the access review process, reviewers(who are users) are contacted to do the review. Based on the reviewers configuration(groups, managers, etc.), the expanded list of users is constructed and those reviewers are contacted to do the review when the review starts. This navigation link returns a collection of those contacted reviewers. Note that this is different from the reviewers property above in the sense that the reviewers property stores the configuration of reviewers whereas the "contactedReviewers" propertys is returns the expanded collection of users based on that configuration. For example, one might configure owners of group G1[with owners U1,U2], members of group G2[with members U3,U4] and users U5,U6 in the "reviewers" property. The "contactedReviewers" property will return [U1,U2,U3,U4,U5,U6]. Supports `$select`.|

Reviewer: ```suggestion
| contactedReviewers   |[accessReviewReviewer](../resources/accessreviewreviewer.md) collection| Returns the collection of reviewers who were contacted to complete this review based on the configuration of `reviewers` and `fallbackReviewers` properties during accessReviewScheduleDefinition creation or updates. Use this property to determine which users were contacted to complete a review. Supports `$select`. Read-only. |
```

| decisions | [accessReviewInstanceDecisionItem](accessreviewinstancedecisionitem.md) collection | Each user reviewed in an [accessReviewInstance](#accessreviewinstance-resource-type) has a decision item representing if their access was approved, denied, or not yet reviewed. |

> | definition |[accessReviewScheduleDefinition](accessreviewscheduledefinition.md) | There is exactly one **accessReviewScheduleDefinition** associated with each instance. It is the parent schedule for the instance, where instances are created for each recurrence of a review definition and each group selected to review by the definition. |

Reviewer 2: ```suggestion
| definition |[accessReviewScheduleDefinition](accessreviewscheduledefinition.md) | This is the parent schedule for the instance. There is exactly one **accessReviewScheduleDefinition** associated with each instance. |
```
